models/stg_reddit_ads__account_report.sql

```diff
@@ -1,9 +1,9 @@
+{{ config(enabled=var('ad_reporting__reddit_ads_enabled', True)) }}
 
 with base as (
 
     select * 
     from {{ ref('stg_reddit_ads__account_report_tmp') }}
-
```

Comment:
Small whitespace removal suggestion.

Please apply the same changes to the other staging models.

Reply:
done